Gems/OpenXRVk/External/OculusOpenXRMobileSDK/README.txt

```diff
@@ -0,0 +1,7 @@
+The Oculus OpenXR Mobile SDK is not included as part of O3DE.
+
+When enabling OpenXRVk Gem, download the SDK and uncompress it in the following folder within the gem:
+OpenXRVk\External\OculusOpenXRMobileSDK
+
+The Oculus OpenXR Mobile SDK can be found in the following link:
+https://developer.oculus.com/downloads/native-android/
```

Comment:
Can you make this a `.md` file do that Markdown is supported?

Reply:
yeah can do, I was following the similar style as https://github.com/aws-lumberyard-dev/o3de/blob/development/Gems/Meshlets/External/Lib/meshoptimizer.txt 

But a `.md` would be better.

Reply:
done